server/src/ormconfig.ts

```diff
@@ -0,0 +1,23 @@
+import { TypeOrmModuleOptions } from '@nestjs/typeorm';
+import { ConfigService } from './packages/config/config.service';
+
+/**
+ * This is duplicated from the ConfigModule code as there's no straightforward way
+ * to get it from main nest application object here
+ */
+const envFilename = '../.env' + (process.env.NODE_ENV ? '.' + process.env.NODE_ENV : '');
+const configService = new ConfigService(envFilename);
+
+const config = {
+  type: configService.get('DB_TYPE'),
```

Comment:
Why are we not using https://github.com/typeorm/typeorm/blob/master/docs/using-ormconfig.md#using-environment-variables ?

Reply:
We added the config service as described here: https://docs.nestjs.com/techniques/configuration

The idea here is that we use an abstraction to whatever configuration strategy we use. Also typeorm uses different environment variable names than we already do so we would need to duplicate those i guess.

As mentioned above, it would be ideal if the typeorm cli could get the ConfigService from a nest application instance (as we do in server/src/main.ts) but i'm not 100% clear on how this could be done.

Reply:
Seems like these env vars have the highest precedence https://github.com/typeorm/typeorm/blob/master/docs/using-ormconfig.md#which-configuration-file-is-used-by-typeorm